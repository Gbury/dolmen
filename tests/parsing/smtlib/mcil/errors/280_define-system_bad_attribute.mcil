(define-system sys :input () :reachable)