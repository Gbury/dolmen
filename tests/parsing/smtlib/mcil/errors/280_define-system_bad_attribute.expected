File "tests/parsing/smtlib/mcil/errors/280_define-system_bad_attribute.mcil", line 1, character 29-39:
1 | (define-system sys :input () :reachable)
                                 ^^^^^^^^^^
Error while parsing a system definition, read the reserved word ':reachable',
      but expected an attribute from the set (:output, :local, :subsys,
      :init, :trans, :inv).
